src/Concordium/Client/Runner.hs

```diff
@@ -2263,7 +2276,7 @@ processBakerCmd action baseCfgDir verbose backend =
              then
                logFatal [[i|Account balance (#{showGtu airAmount}) is lower than the amount requested to be staked (#{showGtu initialStake}).|]]
              else do
-               sendAndMaybeOutputCredentials bakerKeys accountKeysFile outputFile txCfg pl intOpts
+               sendAndMaybeOutputCredentials bakerKeys bakerKeysFile outputFile txCfg pl intOpts
```

Comment:
This `sendAndMaybeOutputCredentials` still outputs keys in plaintext right? It does not ask for password like the key generation command.